src/server/silent_refresh.ts

```diff
@@ -2,7 +2,7 @@ const base: string = process.env.BASE_PATH || '/';
 
 const html = (): string => `
   <!doctype html>
-  <html>
+  <html lang="en" dir="ltr">
```

Comment:
Yes, I understand its current use. However, I thought it would also be beneficial to include it here. This way, users can conduct Lighthouse tests for non-performance-related aspects of the development build :)